test/runtests.jl

```diff
@@ -835,3 +835,41 @@ end
     @test x ≈ [2.3, 2.0]
     @test dx ≈ [1.0]
 end
+
+@testset "Const arg" begin
+	x = Float32[3]
+	w = Float32[1]
+	dw = zero(w)
+
+	function inactiveArg(w, x, cond)
+	   if cond
+		  x = copy(x)
+	   end
+	  @inbounds w[1] * x[1]
+	end
+
+	Enzyme.autodiff(loss, Active, Duplicated(w, dw), Const(x), Const(false))
+
+    @test x ≈ [3.0]
+    @test w ≈ [1.0]
+    @test dw ≈ [3.0]
+
+    # It would be nice to get this right without enabling this
+    Enzyme.API.runtimeActivity!(true)
```

Comment:
Should we always enable this for Julia?

Reply:
Not yet, it currently has a very large perf penalty. We should eventually have a three state, guaranteed active, guaranteed inactive, unknown and default to active (and check at runtime). Currently 1 and 3 are the same